packages/mdc-ripple/_mixins.scss

```diff
@@ -126,15 +154,17 @@
 }
 
 @mixin mdc-ripple-fg-base_($config) {
+  $radius: map-get($config, radius);
+
   @include mdc-ripple-color_($config);
 
   position: absolute;
-  top: 0;
-  left: 0;
-  width: var(--mdc-ripple-fg-size);
-  height: var(--mdc-ripple-fg-size);
-  transform: scale(0);
-  transform-origin: center center;
+  top: calc(50% - #{$radius});
+  left: calc(50% - #{$radius});
+  width: $radius * 2;
+  height: $radius * 2;
+  transform: scale(var(--mdc-ripple-fg-scale));
```

Comment:
In order to make `var()` as bulletproof as possible, consider following the pattern:

```css
transform: scale(0);
transform: scale(var(--mdc-ripple-fg-scale, 0));
```
(replace 0 with whatever default value you deem fit)

The first line gives you a fallback for browsers that don't understand custom properties, whereas the second value in the `var()` function gives you a fallback in case `--mdc-ripple-fg-scale` isn't defined.

Another option is to stick a big a big `@supports` around everything (which to be fair, you may already be doing in JavaScript), but you should at least keep the second parameter to `var()`.